src/coreclr/src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1960,16 +1942,7 @@ void CodeGen::genFMAIntrinsic(GenTreeHWIntrinsic* node)
     // Intrinsics with CopyUpperBits semantics cannot have op1 be contained
     assert(!copiesUpperBits || !op1->isContained());
 
-    if (op3->isContained() || op3->isUsedFromSpillTemp())
```

Comment:
This was merged with the no containment path, since the logic is the same and the containment is handled by the `genHWIntrinsic_R_R_R_RM` call